GameOfLife/GameOfLife/Iteration.cs

```diff
@@ -0,0 +1,156 @@
+﻿namespace GameOfLife
+{
+    public class Iteration
+    {
+        private List<Tuple<int, int>> cellsToDie = new List<Tuple<int, int>>();
+        private List<Tuple<int, int>> cellsToBeBorn = new List<Tuple<int, int>>();
```

Comment:
Private parameters should start with underscore.

Reply:
Added underscores to private parameters.